addons/purchase/views/purchase_views.xml

```diff
@@ -216,13 +216,15 @@
                                                 <label for="product_qty"/>
                                                 <div>
                                                     <field name="product_qty" class="oe_inline"/>
-                                                    <span class="oe_inline">&#160;</span>
-                                                    <field name="product_uom" groups="uom.group_uom" class="oe_inline"/>
+                                                    <field name="product_uom" groups="uom.group_uom" class="oe_inline ml-2"/>
                                                 </div>
+                                                <field name="qty_received_method" invisible="1"/>
+                                                <field name="qty_received" attrs="{'invisible': [('parent.state', 'not in', ('purchase', 'done'))], 'readonly': [('qty_received_method', '!=', 'manual')]}"/>
+                                                <field name="qty_invoiced" attrs="{'invisible': [('parent.state', 'not in', ('purchase', 'done'))]}"/>
                                                 <field name="price_unit"/>
+                                                <field name="taxes_id" widget="many2many_tags" domain="[('type_tax_use', '=', 'purchase')]" options="{'no_create': True}"/>
```

Comment:
Same here, this form is only readonly so you can remove the domain and the options.